components/common/day_picker_navbar/index.tsx

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import { NavbarElementProps } from 'react-day-picker';
+
+interface CustomNavbarProps extends NavbarElementProps {
+    ifSearchDateSuggestion: boolean;
```

Comment:
this should be optional

Reply:
`const Navbar: React.FC<Partial<CustomNavbarProps>> = (navbarProps: Partial<CustomNavbarProps>) => {`
The Partial property makes sure that all fields are optional right?